modules/core/controller.py

```diff
@@ -166,3 +166,9 @@ def get_temp(self, id=None):
             id = self.fermenter_id
         return self.get_sensor_value(int(self.api.cache.get("fermenter").get(id).sensor))
 
+    @cbpi.try_catch(None)
+    def get_temp2(self, id=None):
```

Comment:
probably is better to pass the sensor as a parameter to `get_temp` method with a default to not break existing code...

Reply:
Sorry for may lack of knowledge of pyhon, but how do you pass a member of fermenter as parameter?  
get_temp(self, id=None, sens=sensor):    
   ......get(id).sens ))    do not work